nix/pkgs/extensions/default.nix

```diff
@@ -44,6 +44,19 @@ let
         pname = "upgrade-operator";
       };
     };
+    call-home = rec {
```

Comment:
should `call-home` be part of `exporter`?

Reply:
In my opinion, No as call-home has a separate directory and it is not exposing any data through any HTTP endpoint instead it sends the encrypted JSON to a swarm server every 24 hours. So this data can not be read by Prometheus or in any other way by the user.

Reply:
That's how it currently is (directory structure) but I'm questioning if that's how we really want it to be. 
I thought `exporter` was just a generic name, it's exporting things, and 1 example is `metrics`. As per the nomenclature in the wiki, this is our naming convention: `<classification>-<specific_name>`.
I could see call home here as `exporter.call-home`?

Reply:
Yes, I got your point but this name was decided during the earlier discussions and it is also there in the system impact statement that's why I used this name.